### PR TITLE
Run admin checks and tests on travis

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: admin/Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ serve: build
 
 ${BUILD_DIR}/venv.timestamp:
 	#Create a Python virtual environment.
-	python3 -m venv ${VENV}
+	virtualenv -p python3 ${VENV}
```
Comment (sbrunner) [inline admin/Makefile]: Why this change?

Comment (arnaud-morvan) [reply admin/Makefile]: Because the old version works locally but not on travis, I don't know why.

Comment (sbrunner) [reply admin/Makefile]: Strange I just do a test...

Comment (sbrunner) [reply admin/Makefile]: it should work ...
https://travis-ci.org/camptocamp/c2cgeoportal/jobs/293577574
